radio/src/hal/adc_driver.cpp

```diff
@@ -134,7 +134,8 @@ bool adcRead()
   uint16_t temp[NUM_ANALOGS] = { 0 };
 
   uint8_t first_analog_adc;
-#if defined(RADIO_FAMILY_T16) || defined(PCBNV14)
+
+#if defined(RADIO_FAMILY_T16) || defined(PCBNV14) || defined(RADIO_BOXER)
```

Comment:
This is another... this isn't really T16 or BOXER specific, is it? And if you have a FLYSKY_GIMBALS define, that can be specified for the NV14/EL18, making it completely agnostic.

Reply:
Also here, addressed in latest commit 067cede.